client/src/components/Onboarding/Onboarding.js

```diff
@@ -15,13 +16,16 @@ function Onboarding() {
   }, []);
 
   return (
-    <div>
+    <div className="workbench-wrapper">
```

Comment:
__Question__: Why recreate the Workbench (again, like Manage Account) instead of using the Workbench component to render a section, but __not__ render a link to it?